invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/grid.overrides

```diff
@@ -85,6 +85,10 @@
     padding-top: 5px;
     padding-bottom: 5px;
   }
+
+  .column.searchbar-with-filters{
```

Comment:
```suggestion
  .flex-container {
```
I am trying to remove all the naming which is connected with our elements of the interface, and replace them with the semantic way of naming, which is based on the actual "behaviour", which should reduce the amount of custom CSS - `flex-container` becomes reusable not only for search bar filters. it should be also placed in site.overrides